### DISCUSSION TITLE
Some Question

### DISCUSSION THREAD

**Discussion by hirockstar** [hlf20010508/telegram-onedrive#16] [Q&A] [answered]
with this can we upload large file via URL like this [Link](https://updates.cdn-apple.com/2023FallFCS/fullrestores/042-72841/ACCBB990-9842-4C7D-AD93-AE8E62D76179/iPhone16,2_17.0.3_21A360_Restore.ipsw) .
our account it business account dose it support that?

**Comment by hlf20010508** [comment]
Yes, it can.

<img width="569" alt="image" src="https://github.com/hlf20010508/telegram-onedrive/assets/76218469/6d3aaf71-cfc9-4245-8837-6e0cec2c680b">

But I don't know if it support business account, because I don't have a business account, I've never tried.

**Comment by hlf20010508** [comment]
The OneDrive sdk does support business account.
See [here](https://github.com/OneDrive/onedrive-sdk-python#onedrive-for-business).
You can try to edit [here](https://github.com/hlf20010508/telegram-onedrive/blob/658a808cb23a76975324c3f8ac87035c852581dd/modules/onedrive.py#L48C1).
I can't do any test since I don't have a business account.
You can make a pull request if you succeed.

**Comment by hlf20010508** [comment, marked as answer]
Hey there, business account has been supported, as long as you're not using **educational account without admin**.

**Comment by hirockstar** [comment]
thanks
